internal/namespaces/vpc/v1/custom_private_network.go

```diff
@@ -4,19 +4,53 @@ import (
 	"context"
 
 	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/api/baremetal/v1"
 	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/api/lb/v1"
+	"github.com/scaleway/scaleway-sdk-go/api/rdb/v1"
+	"github.com/scaleway/scaleway-sdk-go/api/redis/v1"
 	"github.com/scaleway/scaleway-sdk-go/api/vpc/v1"
+	"github.com/scaleway/scaleway-sdk-go/api/vpcgw/v1"
 	"github.com/scaleway/scaleway-sdk-go/scw"
 )
 
 func privateNetworkGetBuilder(c *core.Command) *core.Command {
-	type customServer struct {
+	type customInstanceServer struct {
 		ID         string               `json:"id"`
 		Name       string               `json:"name"`
 		State      instance.ServerState `json:"state"`
 		NicID      string               `json:"nic_id"`
 		MacAddress string               `json:"mac"`
 	}
+	type customBaremetalServer struct {
+		ID                 string                 `json:"id"`
```

Comment:
Could you add the IP address within the private network?

Reply:
It's not possible to get the IP address, I added the Vlan number instead